ext/date/php_date.c

```diff
@@ -2263,17 +2263,20 @@ PHPAPI zval *php_date_instantiate(zend_class_entry *pce, zval *object) /* {{{ */
 
 /* Helper function used to store the latest found warnings and errors while
  * parsing, from either strtotime or parse_from_format. */
-static void update_errors_warnings(timelib_error_container *last_errors) /* {{{ */
+static timelib_error_container* update_errors_warnings(timelib_error_container **last_errors) /* {{{ */
```

Comment:
This should likely either return the new value or take `**` and actually assign to it but not both.

Reply:
Better now?

Reply:
Sure :slightly_smiling_face: Not sure though why various tests suddenly fail.